### PR TITLE
Playlist management

### PR DESCRIPTION
**Pull request by praveenreddy1798**
playlist

### CODE REVIEW

File: src/components/CreatePlaylist.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        <div class="modal bg-white position-absolute border-radius-sm pd-sm position-relative">
+          <button
+            class="btn btn-icon position-absolute close-button"
+            onClick={() => {
+              setIsModalVisible(false);
+              setVideo({});
+              setPlaylistName("");
+            }}
+          >
+            <i class="fa fa-2x fa-close secondary-color"></i>
```
Comment (vaibhav18matere) [inline src/components/CreatePlaylist.js]: make it `className` instead of `class`

File: src/components/CreatePlaylist.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+              value={playlistName}
+              onChange={(e) => setPlaylistName(e.target.value)}
+            />
+            <button
+              onClick={() => {
+                if (playlistName) {
+                  addPlaylists(playlistName);
+                  setPlaylistName("");
+                }
+                if (cardType) {
```
Comment (vaibhav18matere) [inline src/components/CreatePlaylist.js:76]: not so sure but wouldn't it just `if-else` ?

File: src/components/PlaylistCard.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  const { title, videos, id } = playlist;
+  const { removePlaylists } = useRemovePlaylists();
+  return (
+    <div class="playlist-card pointer flex gap-p5 pointer border-radius-sm pd-xsm bg-white">
+      <div
+        onClick={() => navigate("/playlist", { state: { playlistId: id } })}
+        class="flex-vertical flex-grow-1 pd-xxsm"
+      >
+        <h4 class="text-align-left">{title}</h4>
+        <h4 class="secondary-color text-align-left mg-t-xsm">
```
Comment (vaibhav18matere) [inline src/components/PlaylistCard.js]: `class` to `className` here as well. also check all other files if in case you are repeating same mistake

**Review by vaibhav18matere — APPROVED** [review]
good to merge, just check these small mistakes

**Review by Koushith — APPROVED** [review]
LGTM. good to merge.

File: src/components/CreatePlaylist.js (hunk 2 above)
Comment (praveenreddy1798) [reply src/components/CreatePlaylist.js:76]: The both are seperate conditions

**Pull request merged**